test/e2e/opm_test.go

```diff
@@ -493,8 +493,6 @@ var _ = Describe("opm", func() {
 	})
 
 	Context("using podman", func() {
-		//FIXME skip podman tests due to failed to add podman to systemd sandbox cgroup
-		return
```

Comment:
interesting -- this isn't how we should be skipping tests in the first place (see the ginkgo `Skip` function).